docs/product/components/popovers.html

```diff
@@ -263,42 +347,92 @@ <h3 class="fs-body2 mb2">Teams</h3>
     </div>
 
     {% header h3 | Hover %}
-    <p class="stacks-copy">If the user doesn’t need to interact with the <em>contents</em> of the popover, it may be appropriate to only show it on hover. This will make popovers feel like a tooltip. To do so, we can rely on stock mouseover and mouseout events from Stimulus.</p>
+    <p class="stacks-copy">If the user doesn’t need to interact with the <em>contents</em> of the popover, it may be appropriate to only show it on hover. This will make popovers feel like a tooltip. To do so, we provide an alternative controller, <code class="stacks-code">s-tooltip</code>, that shows the tooltip only on hover.</p>
 
     <div class="stacks-preview">
 {% highlight html %}
 <button class="s-btn" role="button"
-        aria-controls="popover-example"
-        data-controller="s-popover"
-        data-action="mouseover->s-popover#show mouseout->s-popover#hide"
-        data-s-popover-placement="bottom">
+        aria-describedby="tooltip-example"
+        data-controller="s-tooltip"
+        data-s-tooltip-placement="top-start">
     …
 </button>
-<div class="s-popover"
-     id="popover-example"
-     role="menu"
+<div class="s-popover s-popover__tooltip"
+     id="tooltip-example"
+     role="tooltip"
      aria-hidden="true">
     <div class="s-popover--arrow"></div>
     …
 </div>
 {% endhighlight %}
         <div class="stacks-preview--example bg-black-025">
             <button class="s-btn" role="button"
-                    aria-controls="popover-hover-example"
-                    data-controller="s-popover"
-                    data-action="mouseover->s-popover#show mouseout->s-popover#hide"
-                    data-s-popover-placement="top-start">
+                    aria-describedby="tooltip-example"
+                    data-controller="s-tooltip"
+                    data-s-tooltip-placement="top-start">
                 Hover popover
             </button>
-            <div class="s-popover"
-                id="popover-hover-example"
-                role="menu"
+            <div class="s-popover s-popover__tooltip"
+                id="tooltip-example"
+                role="tooltip"
```

Comment:
Let's load this example up with an _obnoxious_ amount of markup to really drive the point home. How about throwing an `s-tag` and an illustration in there?